tests/common/plugins/tacacs.py

```diff
@@ -14,9 +14,15 @@ def configure_tacacs(ptfhost, duthost, creds, tacacs_server_ip):
     config_facts  = duthost.config_facts(host=duthost.hostname, source="running")['ansible_facts']
     for tacacs_server in config_facts.get('TACPLUS_SERVER', {}):
         duthost.shell("sudo config tacacs delete %s" % tacacs_server)
+
     duthost.shell("sudo config tacacs add %s" % tacacs_server_ip)
     duthost.shell("sudo config tacacs authtype login")
 
+    if config_tacacs_on_switch:
+        duthost.shell("sudo config tacacs add %s" % creds['tacacs_servers'][0])
```

Comment:
What is the reason behind using creds['tacacs_servers'][0], why not keep it as - tacacs_server_ip ?

Reply:
@SuvarnaMeenakshi 
tacacs_server_ip is:
`ptfhost.host.options['inventory_manager'].get_host(ptfhost.hostname).vars['ansible_host']`
There are some situations where we'll want the TACACS server to be an external one and the same for all switches. 
Hence, the IP address of this server is configured in lab.yml file, and we configure the switch to know this TACACS server.